<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="29035" started-at="2020-09-17T20:22:08Z" finished-at="2020-09-17T20:22:37Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="29035" started-at="2020-09-17T20:22:08Z" finished-at="2020-09-17T20:22:37Z">
      <class name="com.technocredits.snapdeal.testscripts.SnapDealLoginTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.technocredits.snapdeal.testscripts.SnapDealLoginTest@3a82f6ef]" name="setup" is-config="true" duration-ms="13973" started-at="2020-09-18T01:52:08Z" finished-at="2020-09-18T01:52:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="loginVerification()[pri:0, instance:com.technocredits.snapdeal.testscripts.SnapDealLoginTest@3a82f6ef]" name="loginVerification" duration-ms="11228" started-at="2020-09-18T01:52:22Z" finished-at="2020-09-18T01:52:33Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[UserName is not present, expected Aavrutia but received Aavruti expected [Aavrutia] but found [Aavruti]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: UserName is not present, expected Aavrutia but received Aavruti expected [Aavrutia] but found [Aavruti]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at com.technocredits.snapdeal.testscripts.SnapDealLoginTest.loginVerification(SnapDealLoginTest.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginVerification -->
        <test-method status="PASS" signature="teardown(org.testng.ITestResult)[pri:0, instance:com.technocredits.snapdeal.testscripts.SnapDealLoginTest@3a82f6ef]" name="teardown" is-config="true" duration-ms="3662" started-at="2020-09-18T01:52:33Z" finished-at="2020-09-18T01:52:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginVerification status=FAILURE method=SnapDealLoginTest.loginVerification()[pri:0, instance:com.technocredits.snapdeal.testscripts.SnapDealLoginTest@3a82f6ef] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.technocredits.snapdeal.testscripts.SnapDealLoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
